src/CommandingOfficers/Bear_Bull.java

```diff
@@ -83,7 +84,7 @@ public void swapD2Ds(boolean setIncome)
     {
       isBull = true;
       if( setIncome )
-        incomePerCity = (int) (incomeBase * BULL_MOD);
+        incomeAdjustment = (int) (incomeBase * BULL_MOD) - gameRules.incomePerCity;
```

Comment:
I woulda just made the mod variables -0.1 and +0.2, then did just 1 + mod on the unit cost, but this is fine too.

Reply:
Hm. Didn't occur to me. Ah well, too late now.